aioairq/utils.py

```diff
@@ -0,0 +1,14 @@
+import re
+
+pattern_number_between_0_and_255 = re.compile(
+    "(2([0-4][0-9]|5[0-5]))|(1[0-9][0-9])|([1-9][0-9])|([0-9])"
+)
+ipv4_pattern = re.compile(
+    "((" + pattern_number_between_0_and_255.pattern + ")\\.){3}" +
+    "(" + pattern_number_between_0_and_255.pattern + ")"
+)
+
+
+def is_valid_ipv4_address(address: str) -> bool:
+    """Checks if address is a valid IPv4 address."""
+    return ipv4_pattern.fullmatch(address) is not None
```

Comment:
I had looked into that and chose to not use it.
The standard library also allows
```py
>>> ipaddress.IPv4Address(4711)
IPv4Address('0.0.18.103')
>>> ipaddress.IPv4Address(b'\x7f\x00\x00\x01')
IPv4Address('127.0.0.1')
```
I wanted to avoid that.

Thinking again about it, nothing really could happen, as Python constructs a valid address air-Q will support.
(Maybe I just had seen the example in the docs about `ipaddress.ip_address(address)` also allowing IPv6 addresses, which air-Q does NOT accept. Could be I confused those two functions)

If you don't mind allowing strange input (integer or bytes object), then I also find the standard library solution better.
What do you think?